src/pp-ga/pp-ga.ts

```diff
@@ -1,3 +1,34 @@
-import './ga';
+import './ga.js';
+import FieldsObject = UniversalAnalytics.FieldsObject;
+import PPSettings from '../PPSettings.interface';
 
-ga('create', 'UA-123054125-2', 'auto');
+const GA_ID_PROD = 'UA-123054125-1';
+const GA_ID_DEV = 'UA-123054125-2';
+
+export function init() {
+  ga('create', PP_SETTINGS.DEV ? GA_ID_DEV : GA_ID_PROD);
+  if (PP_SETTINGS.DEV) {
+    ga('set', 'checkProtocolTask', () => { /* nothing */ });
+  }
+}
+
+function sendEvent(fieldsObject: FieldsObject) {
+  ga('send', 'event', fieldsObject);
+}
+
+export function extensionInstalled() {
+  sendEvent({ eventCategory: 'Extension', eventAction: 'install', eventLabel: 'ExtensionInstalled' });
+}
+
+export function extensionUpgradedFrom(previousVersion) {
+  sendEvent({
+    eventCategory: 'Extension', eventAction: 'upgrade', eventLabel: 'ExtensionUpgradedFrom ' + previousVersion,
+  });
+}
+
+// This need to be in fact implemented on our site frontend, not in the extension
```

Comment:
I haven't tested the concept yet, but as far as I understand it, this must be external URL, because this is last step in the process of uninstallation and no piece of our code (so no extension based URLs) is there anymore.